main.go

```diff
@@ -82,29 +80,26 @@ func main() {
 	})
 
 	log := logger.NewUPPLogger(*appSystemCode, *logLevel)
+	log.WithFields(map[string]interface{}{
+		"CACHE_DURATION": *cacheDuration,
+		"NEO_URL":        *neoURL,
```

Comment:
nitpick(non-blocking): Can we also add the `port` here for better consistency with the other migrated services?
```suggestion
		"NEO_URL":        *neoURL,
		"PORT":        *port,
```